srcpkgs/cpux/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'cpux'
+pkgname=cpux
+version=4.4.0
+revision=1
+wrksrc=CPU-X-${version}
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release"
+hostmakedepends="pkg-config automake cmake gtk+3 libcpuid libtool ncurses procps-ng "
+makedepends="pkg-config automake cmake gtk+3 gtk+3-devel libcpuid libcpuid-devel libtool ncurses procps-ng procps-ng-devel"
```

Comment:
- why is automake needed?
- cmake doesn't need to be specified
- pkg-config should only be a hostmakedepend
- only put `-devel` packages in makedepends
- missing `pciutils-devel`, `glfw-devel` (probably also need `configure_args="-DWITH_VULKAN=OFF"`)
- non-devel versions of makedepends do not need to be specified anywhere